### PR TITLE
automatically do a minor release each time the LMS releases

### PR DESCRIPTION
**Pull request by dlockhart**
Switching core to use the new `MINOR_RELEASE_WITH_LMS` functionality.

### CODE REVIEW

**Comment by dlockhart** [comment]
TODO: I need to make the `RALLY_API_KEY` secret available to this org

**Comment by svanherk** [comment]
This is technically an issue with every repo that uses `semantic-release`, right?  And moving to CodeArtifact and NPM removes a previous workaround where you could point to a specific tag using GitHub refs directly...
I'm wondering if this should just be added to the semantic-release action, and you "opt-in" by providing a location to your `.lmsrelease` file 🤔

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -27,6 +27,26 @@ jobs:
...
+      - name: Update Repo's Last LMS Release
+        if: steps.repo-lms-version.outputs.VALUE != steps.get-lms-version.outputs.LMS_VERSION
+        run: |
+          echo "Updating last LMS release..."
+          echo "${{ steps.get-lms-version.outputs.LMS_VERSION }}" > .lmsrelease
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+          git add .lmsrelease
+          git commit -m "feat: updating tracked LMS release version [skip ci]"
+          git push --follow-tags --repo=https://${{ inputs.GITHUB_TOKEN }}@github.com/${{ github.repository }}
```
Comment (svanherk) [inline .github/workflows/release.yml:47]: Should this be `${{ secrets.D2L_GITHUB_TOKEN }}` or `$GITHUB_TOKEN`?

Comment (dlockhart) [reply .github/workflows/release.yml:47]: Yeah probably one or the other -- they should both be equivalent in this context. Not sure how this worked in my version of this that I tested with.

**Review by svanherk — APPROVED** [review]
So in theory, you could try releasing this with no `fix:` or `feat:` and it should do a minor bump...

**Pull request merged**